django2pydantic/handlers/json.py

```
@@ -1,24 +1,24 @@
 """Handler for JSON fields."""
 
-from typing import override
+from typing import Annotated, override, reveal_type
```

Ruff (0.8.2)

`typing.Annotated` imported but unused

Remove unused import

(F401)